site-configs.ts

```diff
@@ -4,42 +4,50 @@ import { SiteConfig } from "./site-configs.types";
 export default (): SiteConfig[] => [
```

Comment:
Out of scope: We should have the same strategy for site configs in Demo as in the Starter (configs in a separate folder)

Reply:
Yes, I will also make the changes I made in the starter (https://github.com/vivid-planet/comet-starter/pull/283) in Demo. But I first want to focus on having a good solution in starter since the basis there is better

Reply:
Developing in demo is easier as you can make changes (like this one) along with it.

Imho demo-first approach should be preferred